src/components/Tabs/tests/Tabs.test.tsx

```diff
@@ -108,38 +108,37 @@ describe('<Tabs />', () => {
         {...tabs[1], panelID: 'panel-2'},
       ];
       const content = <p>Panel contents</p>;
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
         <Tabs {...mockProps} tabs={panelIDedTabs}>
           {content}
         </Tabs>,
       );
 
-      panelIDedTabs.forEach((tab, index) => {
-        expect(wrapper.find('ul').find(Tab).at(index).prop('panelID')).toBe(
-          tab.panelID,
-        );
+      panelIDedTabs.forEach((tab) => {
+        expect(wrapper.find('ul')!).toContainReactComponent(Tab, {
+          panelID: tab.panelID,
+        });
       });
     });
 
     it('uses an auto-generated panelID if none is provided', () => {
       const content = <p>Panel contents</p>;
-      const wrapper = mountWithAppProvider(
-        <Tabs {...mockProps}>{content}</Tabs>,
-      );
+      const wrapper = mountWithApp(<Tabs {...mockProps}>{content}</Tabs>);
 
       tabs.forEach((_, index) => {
-        const panelID = wrapper.find('ul').find(Tab).at(index).prop('panelID');
+        const panelID = wrapper.find('ul')?.findAll(Tab)[index].prop('panelID');
```

Comment:
```suggestion
        const panelID = wrapper.find('ul')!.findAll(Tab)[index].prop('panelID');
```

Just a suggestion for future tests. I find it's easier to debug if you use TS `!` vs `?`. Because the optional chaining will fail silently and you'll have to do a bit more work to find out what's going wrong in the test. Where as if you use `!` you'll see right away that your test can't find `ul`